templates/gsl.vcxproj

```diff
@@ -241,6 +241,14 @@ function is_product_library(product)
     return (my.product.prefix = "lib")
 endfunction
 
+function is_compiler_ctp(product)
+    define my.product = is_compiler_ctp.product
+    return is_product_library(my.product) |\
+        (is_product_binary(my.product) & !is_option_tests(my.product)) |\
+        (is_product_binary(my.product) & is_option_tests(my.product) &\
```

Comment:
I can't see a reason for a libconsensus exception here. It should halve been the same as the other libs.

Reply:
A removal of the condition and recompilation of libbitcoin-consensus results in the following error(s):
```
z:\git\libbitcoin-consensus\src\clone\serialize.h(40): error C2146: syntax error : missing ';' before identifier 'deser
ialize_type' (..\..\..\..\test\consensus__verify_flags_to_script_flags.cpp) [z:\git\libbitcoin-consensus\builds\msvc\vs
2013\libbitcoin-consensus-test\libbitcoin-consensus-test.vcxproj]
z:\git\libbitcoin-consensus\src\clone\serialize.h(40): error C4430: missing type specifier - int assumed. Note: C++ doe
s not support default-int (..\..\..\..\test\consensus__verify_flags_to_script_flags.cpp) [z:\git\libbitcoin-consensus\b
uilds\msvc\vs2013\libbitcoin-consensus-test\libbitcoin-consensus-test.vcxproj]
z:\git\libbitcoin-consensus\src\clone\serialize.h(814): error C2144: syntax error : 'bool' should be preceded by ';' (.
.\..\..\..\test\consensus__verify_flags_to_script_flags.cpp) [z:\git\libbitcoin-consensus\builds\msvc\vs2013\libbitcoin
-consensus-test\libbitcoin-consensus-test.vcxproj]
z:\git\libbitcoin-consensus\src\clone\serialize.h(814): error C4430: missing type specifier - int assumed. Note: C++ do
es not support default-int (..\..\..\..\test\consensus__verify_flags_to_script_flags.cpp) [z:\git\libbitcoin-consensus\
builds\msvc\vs2013\libbitcoin-consensus-test\libbitcoin-consensus-test.vcxproj]
z:\git\libbitcoin-consensus\src\clone\serialize.h(818): error C2144: syntax error : 'bool' should be preceded by ';' (.
.\..\..\..\test\consensus__verify_flags_to_script_flags.cpp) [z:\git\libbitcoin-consensus\builds\msvc\vs2013\libbitcoin
-consensus-test\libbitcoin-consensus-test.vcxproj]
z:\git\libbitcoin-consensus\src\clone\serialize.h(818): error C4430: missing type specifier - int assumed. Note: C++ do
es not support default-int (..\..\..\..\test\consensus__verify_flags_to_script_flags.cpp) [z:\git\libbitcoin-consensus\
builds\msvc\vs2013\libbitcoin-consensus-test\libbitcoin-consensus-test.vcxproj]
msbuild /verbosity:minimal /p:Platform=x64 /p:Configuration=StaticRelease libbitcoin-consensus.sln failed.
Building libbitcoin-consensus failed.
```

Reply:
Looks like it wasn't an oversight, just forgot about this issue.